tests/parser/SaltTableTests.cpp

```diff
@@ -53,26 +56,40 @@ BOOST_AUTO_TEST_CASE( Brine ) {
         "1 1/\n"
         "\n"
         "PVTWSALT\n"
-        " 1000/\n"
+        " 1000 0.0/\n"
         " 0 1 2 3 4 \n"
         " 10 11 12 13 14/\n"
-        " \n"
+        "\n"  
+        "RWGSALT\n"
+        " 300 0.0 0.00013/ \n"
+        " 600 0.5 0.000132/ \n"
+        "/ \n"  
+        "PERMFACT\n"
```

Comment:
The `PERMFACT` keyword as specified in json does not have a `size:` setting - that implies that it should have default size (i.e. number of records) behavior, which again means that the whole keyword should have a slash termination - i.e. something like:
```
PERMFACT
   0    0
   0.5 0.5
   1    1
   1.5  1.5  /  -- Terminate the first record
/                   -- Terminate the whole keyword.
```
I do not know how this should be used - if it is *always* only one record you could add `"size" : 1` to the `PERMFACT` json scheme?

Reply:
Thanks for catching it. Multiples tables should be allowed, I have added the extra slash.